src/Components/Footer/Contact.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import styles from './styles';
+
+function Contact() {
+  return (
+    <View style={[styles.col, styles.right]}>
+      <Text style={[styles.text, styles.heading]}>Contact Us</Text>
+      <Text style={[styles.text, styles.description]}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc interdum
```

Comment:
no `Lorem ipsum dolor` will be merge. @sakshi1499 do we have content team issue for that?

Reply:
I'll create it by today...

Reply:
@annabauza no content team issue for footer has been created yet. Thank you @sonal240

Reply:
@tichnas the content has been merged